src/test/java/uk/gov/justice/digital/delius/service/ConvictionServiceTest.java

```diff
@@ -592,6 +687,11 @@ private Event aCustodialEvent(String prisonerNumber, boolean active, String cust
                 .disposal(disposal)
                 .build();
     }
+
+    private DisposalType aNcDisposalType() {
+        return DisposalType.builder().sentenceType("NC").build();
```

Comment:
# Also pull the code out of the DisposalType entity?

Reply:
Will create as a constant in the test class as it's only needed as an example in the test class to support the creation of a conviction/disposal that can be transformed into a dto. Sentence type is not a ref type in it's own right.